[2024-06-03T02:35:25.892+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T02:35:25.921+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load scheduled__2024-06-02T00:00:00+00:00 [queued]>
[2024-06-03T02:35:25.933+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load scheduled__2024-06-02T00:00:00+00:00 [queued]>
[2024-06-03T02:35:25.934+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-03T02:35:25.949+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load> on 2024-06-02 00:00:00+00:00
[2024-06-03T02:35:25.958+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=218) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-03T02:35:25.960+0000] {standard_task_runner.py:63} INFO - Started process 233 to run task
[2024-06-03T02:35:25.958+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load', 'scheduled__2024-06-02T00:00:00+00:00', '--job-id', '217', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpm_q4uhei']
[2024-06-03T02:35:25.961+0000] {standard_task_runner.py:91} INFO - Job 217: Subtask load
[2024-06-03T02:35:25.975+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-03T02:35:26.015+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.load scheduled__2024-06-02T00:00:00+00:00 [running]> on host c5ef054da447
[2024-06-03T02:35:26.113+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-06-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-02T00:00:00+00:00'
[2024-06-03T02:35:26.115+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T02:35:26.138+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/google/cloud/firestore_v1/base_collection.py:302 UserWarning: Detected filter using positional arguments. Prefer using the 'filter' keyword argument instead.
[2024-06-03T02:35:36.119+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T02:35:36.120+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T02:35:36.131+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load, run_id=scheduled__2024-06-02T00:00:00+00:00, execution_date=20240602T000000, start_date=20240603T023525, end_date=20240603T023536
[2024-06-03T02:35:36.159+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T02:35:36.174+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T02:35:36.177+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
